tests/Get-DbaDbCompatibility.Tests.ps1

```diff
@@ -14,14 +14,18 @@ Describe "$CommandName Unit Tests" -Tag 'UnitTests' {
 }
 
 Describe "$commandname Integration Tests" -Tags "IntegrationTests" {
+    BeforeAll {
+        $server = Connect-DbaInstance -SqlInstance $script:instance1
+        $compatibilityLevel = $server.Databases['master'].CompatibilityLevel
+    }
     Context "Gets compatibility for multiple databases" {
         $results = Get-DbaDbCompatibility -SqlInstance $script:instance1
         It "Gets results" {
             $results | Should Not Be $null
         }
         Foreach ($row in $results) {
-            It "Should return Compatiblity level of Version100 for $($row.database)" {
-                $row.Compatibility | Should Be "Version100"
+            It "Should return correct compatiblity level for $($row.database)" {
```

Comment:
compat*I*bility here. may sound nitpicky, but given we're reviewing, we may fix "formalities" as well.